hr_employee_cost_history/models/hr_employee_timesheet_cost_history.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2024 Moduon Team S.L. <info@moduon.team>
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl).
+
+from odoo import fields, models
+
+
+class HrEmployeeTimesheetCostHistory(models.Model):
+    _name = "hr.employee.timesheet.cost.history"
+    _description = "Employee Timesheet Cost History"
+    _order = "create_date DESC"
+
+    employee_id = fields.Many2one(
+        comodel_name="hr.employee",
+        string="Employee",
+    )
+    currency_id = fields.Many2one(
+        comodel_name="res.currency",
+        string="Currency",
+    )
+    timesheet_cost = fields.Monetary(currency_field="currency_id")
```

Comment:
suggestion: rename this field, so it fits better the name it will have in Odoo 16.0, where [there's been a refactor on this model](https://github.com/odoo/odoo/commit/a3eec47f35853a6fa88f94885491f14e7ff94ef6#diff-bf0d01d99e8b400e9a1e3a37eaba5f2d6851a9c3bf3ee990a38f51455fc991d2).

```suggestion
    hourly_cost = fields.Monetary(currency_field="currency_id")
```